### PR TITLE
Converts Poppies, Lilies, Geraniums, and Rainbow Bunch flowers to GAGS

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

This PR converts the rainbow bunch flowers from using primitive greyscale techniques to using GAGS. And because they use the same sprite as the poppy family of flowers - Poppies, lilies, and geraniums have been converted too.

image

This PR also moves all the worn hydroponics sprites out of the general `head.dmi` worn file into their own dmi.

## Why It's Good For The Game

Rainbow bunch used to use greyscale, but not GAGS, and they looked wack! Now, they look bright and beautiful.

## Changelog
:cl: Melbert
add: GAGS flowers (Rainbow bunch flowers now look prettier!)
refactor: Rainbow bunch flowers now use GAGS instead of their old method of greyscaling
/:cl:

### CODE REVIEW

**Comment by CRITAWAKETS** [comment]
anytime i see a screenshot with pixel distortion, my ancestors scream from within their chambers in my mind

**Review by Twaticus — APPROVED** [review]

**Comment by maxymax13** [comment]
why do your screenshots look like they were scaled down and up in paint 7 times

**Comment by TheBonded** [comment]
> why do your screenshots look like they were scaled down and up in paint 7 times

this is how 2d spessmans looks zoomed in

**Comment by MrMelbert** [comment]
> 
> 
> why do your screenshots look like they were scaled down and up in paint 7 times

because they were scaled down and up in paint 7 times

File: code/modules/hydroponics/grown/rainbow_bunch.dm [hunk 1]
```diff
@@ -33,48 +32,51 @@
...
-			color = "#1C1C1C"
-			reagents.add_reagent(/datum/reagent/colorful_reagent/powder/black, 3)
-			dye_color = DYE_BLACK
-			desc += " This one is in a midnight black color."
-		if(8)
-			color = "#FFFFFF"
-			reagents.add_reagent(/datum/reagent/colorful_reagent/powder/white, 3)
-			dye_color = DYE_WHITE
-			desc += " This one is in a pure white color."
+	if(!greyscale_colors)
```
Comment (Mothblocks) [inline code/modules/hydroponics/grown/rainbow_bunch.dm:40]: Early-return.

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Comment by MrMelbert** [comment]
unrelated failure /shrug

**Pull request merged**